src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1446,7 +1464,7 @@ MulticopterPositionControl::task_main()
 							float i = _params.thr_min;
 
 							if (reset_int_z_manual) {
-								i = _manual.z;
+								i = _params.thr_hover;
```

Comment:
@kd0aij I have a final question about this. Do you feel any difference in the transition from manual to alt ctl to before? E.g. for quads which have a considerable lower / higher hover thrust value? Theoretically, we could save the parameter value to this parameter.